packages/fxa-auth-server/lib/db.js

```diff
@@ -1358,6 +1358,58 @@ module.exports = (
     return this.pool.del(SAFE_URLS.deleteRecoveryKey, { uid });
   };
 
+  SAFE_URLS.createAccountSubscription = new SafeUrl(
+    '/account/:id/subscriptions/:subscriptionId',
```

Comment:
Typo: this `:id` in the safe URL should be `:uid` to match the argument to `pool.put` (or vice versa).

Reply:
Oof yeah, I made that typo all over the place :/